requirements_dev.txt

```diff
@@ -1,14 +1,17 @@
 pytest>=6.0
+pytest-runner
```

Comment:
`pytest-runner` is obsolete. Can likely be removed from cookiecutter.

Reply:
removed